ticket_facts.view.lkml

```diff
@@ -0,0 +1,277 @@
+      description: "Yes, if inbound email was solved at first agent touch"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${ticket_source} LIKE 'Inbound Email%'
+         AND ${status} IN ('solved', 'closed')
+         AND ${is_ticket_merged} = false
+         AND (${number_outbound_emails} + ${number_outbound_calls}) = 1
+         AND ${is_parent_to_merged_tickets} = false ;;
+    }
+
+    dimension: is_eligible_for_first_touch_resolution_email {
+      description: "Yes, if inbound email was solved and has outbound activity"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${ticket_source} LIKE 'Inbound Email%'
+          AND ${status} IN ('solved', 'closed')
+          AND ${is_ticket_merged} = false
+          AND (${number_outbound_emails} + ${number_outbound_calls}) >= 1 ;;
+    }
+
+    dimension: is_first_touch_resolution_phone_call {
+      description: "Yes, if inbound phone call was solved at first agent touch"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${ticket_source} IN ('Inbound Call','Inbound Voicemail')
+           AND ${status} IN ('solved', 'closed')
+           AND ${is_ticket_merged} = false
+           AND ${number_inbound_calls} = 1
+           AND (${number_outbound_emails} + ${number_outbound_calls}) = 0
+           AND ${is_parent_to_merged_tickets} = false ;;
+    }
+
+    dimension: is_eligible_for_first_touch_resolution_phone_call {
+      description: "Yes, if inbound phone call was solved"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${ticket_source} IN ('Inbound Call','Inbound Voicemail')
+           AND ${status} IN ('solved', 'closed')
+           AND ${is_ticket_merged} = false
+           AND ${number_inbound_calls} >= 1 ;;
+    }
+
+    dimension: number_outbound_emails {
+      description: "Number of outbound emails associated with ticket"
+      group_label: "Activity Counts"
+      type: number
+      sql: ${TABLE}.number_outbound_emails ;;
+    }
+
+    dimension: has_outbound_emails {
+      description: "Yes, if ticket has associated outbound emails"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${number_outbound_emails} > 0 ;;
+    }
+
+    dimension: number_internal_comments {
+      description: "Number of internal comments associated with ticket"
+      group_label: "Activity Counts"
+      type: number
+      sql: ${TABLE}.number_internal_comments ;;
+    }
+
+    dimension: has_internal_comments {
+      description: "Yes, if ticket has internal comments created"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${number_internal_comments} > 0 ;;
+    }
+
+    dimension: is_parent_to_merged_tickets {
+      description: "Yes, if the ticket is the parent to other merged tickets"
+      group_label: "Ticket Details"
+      type: yesno
+      sql: ${TABLE}.number_merged_tickets > 0 ;;
+    }
+
+    dimension: is_ticket_merged {
```

Comment:
Perhaps `is_merged_into_another_ticket` is more clear?

Reply:
done